<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QMainWindow" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="settingpic">
   <property name="styleSheet">
    <string notr="true">#settingpic{border-image:url(:/gui/settingWindow/bk.jpg)}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;楷体&quot;;
font-weight: bold; 
color:rgb(46, 99, 45)</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnReturn {
    border: none;
    }

QPushButton#btnReturn:hover {
	icon: url(:/gui/aboutWindow/2-.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resource/resource.qrc">
      <normaloff>:/gui/aboutWindow/2.png</normaloff>:/gui/aboutWindow/2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMusic">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resource/resource.qrc">
      <normaloff>:/gui/settingWindow/1.png</normaloff>:/gui/settingWindow/1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSound">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resource/resource.qrc">
      <normaloff>:/gui/settingWindow/3.png</normaloff>:/gui/settingWindow/3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_music">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>音乐</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_sound">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>音效</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
